BiometricLocker.xcodeproj/xcshareddata/xcschemes/BiometricLocker.xcscheme

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "0930"
+   version = "1.3">
+   <BuildAction
+      parallelizeBuildables = "YES"
+      buildImplicitDependencies = "YES">
+      <BuildActionEntries>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "YES"
+            buildForProfiling = "YES"
+            buildForArchiving = "YES"
+            buildForAnalyzing = "YES">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "9FAE071C20849AF70018B8E8"
+               BuildableName = "BiometricLocker.framework"
+               BlueprintName = "BiometricLocker"
+               ReferencedContainer = "container:BiometricLocker.xcodeproj">
+            </BuildableReference>
+         </BuildActionEntry>
+      </BuildActionEntries>
+   </BuildAction>
+   <TestAction
+      buildConfiguration = "Debug"
+      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
+      selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      shouldUseLaunchSchemeArgsEnv = "YES">
+      <Testables>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "9FAE072D2084A4E80018B8E8"
+               BuildableName = "BiometricLockerTests.xctest"
+               BlueprintName = "BiometricLockerTests"
+               ReferencedContainer = "container:BiometricLocker.xcodeproj">
```

Comment:
can you make sure that the box for `gather code coverage` is checked on the testing scheme please? I think it should show up right around here but I'm not seeing it anywhere in this file

Reply:
I assume that this no longer breaks lose all kinds of hell, like it did last year?

Reply:
I haven't had any issues with it myself, but feel free to uncheck it if it causes problems

Reply:
Wasn't able to submit builds to iTunesConnect with it on, and carthage had no option to disable it before building it. Was hell.

Reply:
Oh that's damned weird. I have not tried that with schemes intended to build stuff from Carthage - maybe just check it on the scheme for the test app then.

Reply:
🤞🏽 We can test and see!

Reply:
And it looks like it works fine now. 👍 Leaving coverage in.